src/API/Base.php

```diff
@@ -399,55 +458,35 @@ public static function get_merchant( $merchant_id ) {
 
 
 	/**
-	 * Creates a merchant for the authenticated user or returns the existing one.
-	 *
-	 * @return mixed
-	 */
-	public static function maybe_create_merchant() {
-
-		$args = array(
-			'display_name'                      => apply_filters( 'pinterest_for_woocommerce_default_merchant_name', esc_html__( 'Auto-created by Pinterest for WooCommerce', 'pinterest-for-woocommerce' ) ),
-			'return_merchant_if_already_exists' => true,
-		);
-
-		return self::make_request(
-			add_query_arg( $args, 'commerce/product_pin_merchants/' ),
-			'POST'
-		);
-	}
-
-
-	/**
-	 * Adds the merchant's feed using the given arguments.
+	 * Creates a merchant for the authenticated user or updates the existing one. On success the Merchant ID is returned instead of the full merchant object.
 	 *
-	 * @param string $merchant_id The merchant ID the feed belongs to.
-	 * @param array  $args        The arguments to be passed to the API request.
+	 * @param array $args Payload to be sent to the request.
 	 *
 	 * @return mixed
 	 */
-	public static function add_merchant_feed( $merchant_id, $args ) {
```

Comment:
Just a note that we should be cautious about removing functions declared as `public`. While it's currently unlikely that anyone else is writing code dependent on ours, this is how compatibility breaks happen.

If we need to remove some functions/methods, we should consider having some way to mark these as deprecated. Even better is to ensure the only methods marked `public` absolutely must be `public`.

I think I'll open an issue so what we can more comprehensively determine how to handle this.